<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tableViewButton">
        <property name="text">
         <string>Table View</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Injection</string>
         </property>
         <item>
          <property name="text">
           <string>Injector Flow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Injector Dead Time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Air/Fuel Ratio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Volumetric Efficiency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Timing</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Ignition</string>
         </property>
         <item>
          <property name="text">
           <string>Timing Advance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Timing Advance Increment Step</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Timing Advance Decrement Step</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Crank</string>
         </property>
         <item>
          <property name="text">
           <string>Injector Pulsewidth</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Idle</string>
         </property>
         <item>
          <property name="text">
           <string>Target RPM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Air/Fuel Ratio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ignition Timing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ignition Timing Offset</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Actuator</string>
          </property>
          <item>
           <property name="text">
            <string>PID Settings</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PWM Frequency</string>
           </property>
          </item>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>horizontalSpacer_2</zorder>
         <zorder>pwmFrequencyComboBox</zorder>
         <zorder>pushButton</zorder>
         <zorder>radioButton</zorder>
         <zorder>checkBox</zorder>
         <zorder>comboBox_2</zorder>
         <zorder>lineEdit</zorder>
         <zorder>spinBox</zorder>
         <zorder>doubleSpinBox</zorder>
         <zorder>progressBar</zorder>
         <zorder>horizontalSpacer_2</zorder>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Injection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ignition</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Idle</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="dataTreeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="indentation">
           <number>10</number>
          </property>
          <property name="itemsExpandable">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unit</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Idle</string>
           </property>
           <item>
            <property name="text">
             <string>Target RPM</string>
            </property>
            <property name="text">
             <string>850</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RPM Delta</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Timing Offset</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="text">
             <string>BTDC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PWM Duty</string>
            </property>
            <property name="text">
             <string>28</string>
            </property>
            <property name="text">
             <string>%</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
